css/selectors/matches.json

```diff
@@ -8,60 +8,95 @@
           "support": {
             "chrome": [
               {
-                "version_added": "66"
+                "version_added": "66",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "#enable-experimental-web-platform-features",
+                    "value_to_set": "enabled"
+                  }
+                ],
+                "notes": [
+                  "It has issues with combinators (<a href='https://crbug.com/842157'>Chromium bug 842157</a>)"
```

Comment:
Change to `"Has issues with combinators (see <a href='https://crbug.com/842157'>bug 842157</a>)."`.